<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,10">
    <item>
     <widget class="QLabel" name="title_label">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Spritesheet XML Generator for Friday Night Funkin'</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="charname_input_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="charname_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Character Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="charname_textbox">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="myTabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="xmlframes_tab">
       <attribute name="title">
        <string>XML from Frames</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
        <item>
         <widget class="QFrame" name="container_frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
           <item>
            <widget class="QScrollArea" name="frames_area">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="sprite_frame_content">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>990</width>
                <height>461</height>
               </rect>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="controls_frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,2">
              <item>
               <widget class="QCheckBox" name="cliptobbox_check">
                <property name="text">
                 <string>Clip to bounding box</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="posename_btn">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>Set Animation (Pose) Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="generatexml_btn">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>Generate XML</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="icongrid_tab">
       <attribute name="title">
        <string>Add Icons to Icon-grid</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QFrame" name="container_frame_icongrid">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5" stretch="10,1">
           <item>
            <widget class="QScrollArea" name="scrollArea_2">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>990</width>
                <height>451</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="icongrid_holder_label">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="controls_frame_icongrid">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,0,0,1,1">
              <item>
               <widget class="QPushButton" name="uploadicongrid_btn">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>Upload Icon-grid</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="uploadicons_btn">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>Upload Icons</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="iconselected_label">
                <property name="text">
                 <string>No. of
icons selected:
0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="tip_label">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Tip: Use ctrl+i and ctrl+o to zoom in or out respectively</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="zoom_label">
                <property name="text">
                 <string>Zoom:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="generateicongrid_btn">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>Generate New
Icon-grid</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1066</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_existing_Spritesheet_and_XML"/>
    <addaction name="actionImport_Images"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport_existing_Spritesheet_and_XML">
   <property name="text">
    <string>Import existing Spritesheet and XML</string>
   </property>
  </action>
  <action name="actionImport_Images">
   <property name="text">
    <string>Import Images</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
